content/docs/reference/clouds/gcp/_index.md

```diff
@@ -32,12 +32,32 @@ In addition to the tutorial, several interesting examples are available complete
 
 ## Example
 
+{{< langchoose >}}
+
 ```javascript
 const gcp = require("@pulumi/gcp")
 
 const bucket = new gcp.storage.Bucket("my-bucket");
 ```
 
+```typescript
+const gcp = require("@pulumi/gcp")
```

Comment:
While this certainly works in TypeScript, it'd be more idiomatic to do:

```suggestion
import * as gcp from "@pulumi/gcp";
```

Reply:
Yes - we should take this change to be consistent with the style we use in all our other TypeScript code snippet.